aiven/client/cli.py

```diff
@@ -4634,20 +4655,20 @@ def user__create(self):
         self._write_auth_token_file(token=result["token"], email=self.args.email)
 
     @arg.json
-    def user__info(self):
+    def user__info(self) -> None:
         """Show current user info"""
         result = self.client.get_user_info()
         layout = [["user", "real_name", "state", "token_validity_begin", "projects", "auth"]]
         self.print_response([result], json=self.args.json, table_layout=layout)
 
-    def _write_auth_token_file(self, token, email):
+    def _write_auth_token_file(self, token: str, email: str) -> None:
         with self._open_auth_token_file(mode="w") as fp:
             fp.write(jsonlib.dumps({"auth_token": token, "user_email": email}))
             aiven_credentials_filename = fp.name
         os.chmod(aiven_credentials_filename, 0o600)
         self.log.info("Aiven credentials written to: %s", aiven_credentials_filename)
 
-    def _open_auth_token_file(self, mode="r"):
+    def _open_auth_token_file(self, mode: str = "r") -> IO:
```

Comment:
Probably `TextIO`

Reply:
actually, depends on what `mode` the file is opened in. Here it is default to read, but it could be used with other mode too, so the return type is not necessarily just `TextIO`. I thought we could just use `IO` to be more generic